### PR TITLE
Assert that transaction is executed in the expected term

### PR DESCRIPTION
**Pull request by majecty**
Currently, some tests are sometiles failed. I suspect that term is
changed unintentionally. This error log will help to find the code
that needs more timeout.

### CODE REVIEW

**Comment by foriequal0** [comment]
If you want to assert that a transaction is executed in a certain term, it would be better to check directly (by getting termId of tx's block), rather than relying on uncertain proving method.

**Comment by majecty** [comment]
@foriequal0 I updated the PR.

File: test/src/e2e.dynval/setup.ts [hunk 1]
```diff
@@ -464,3 +464,36 @@ export function setTermTestTimeout(
...
 }
+
+export async function termThatIncludeTransaction(
+    sdk: SDK,
+    txHash: H256
+): Promise<number> {
+    // Diable lint rule to extract let and const variable from Promise.all.
+    // tslint:disable-next-line:prefer-const
+    let [termMetadata, transaction] = await Promise.all([
+        stake.getTermMetadata(sdk),
```
Comment (foriequal0) [inline test/src/e2e.dynval/setup.ts]: You can utilize `blockNumber` parameter

Comment (majecty) [reply test/src/e2e.dynval/setup.ts]: Good!
I changed the code to use the blockNumber parameter.

**Pull request merged**